### ISSUE TITLE
The scrollable images to the right doesn't have their labels visible

### ISSUE THREAD

**Issue by codeGurhans** [Agarwalvidu/ArdhNaariShakti_Taara_Frontend#37]
Hey @Agarwalvidu 

I'd love to contribute! I noticed some small issues with text readability that could be improved for better UX and accessibility.

The yellow text (e.g., “Sport” and “Fun”) on the scrollable images is difficult to read due to low contrast against the bright background. This affects both aesthetics and accessibility.

Why it's a problem?

1. Poor contrast between text and image leads to low readability.
2. May violate WCAG 2.1 Accessibility Guidelines which require a minimum contrast ratio.
3. Users may miss key context or categories if they can’t read the text clearly.

Fixes:

1. Add a semi-transparent background behind the text..
2. Use a text shadow to enhance readability.
3. Change the text color to a darker(instead of yellow used in this case) or contrasting tone, such as white or black depending on the image.
4.Use a blurred overlay behind the text or dim the image slightly using filter: brightness(0.7).

Please assign me this task to me. Also add the labels "gssoc25" and "level" according to you. 

<img width="1244" height="374" alt="Image" src="https://github.com/user-attachments/assets/a7cf3b44-6ce7-43bb-9d2b-daedb220b31e" />
